content/docs/people/reading-material/onboarding-roles.md

```diff
@@ -1,56 +1,57 @@
 ---
 title: Onboarding Roles
-description: >
+description: |
   This page explains the different roles during the onboarding of new joiners.
 confidentiality: public
 ---
-
 A quick overview of the entire onboarding can be found [here](https://miro.com/app/board/uXjVOIhZj6I=/).
 
 ## Recruiter
+
 ### Who's that?
 
-The person who recruited the new joiner. The Recruiter provides the overall structure for the onboarding process (github tickets, main accounts) and is in general the fallback in case the buddy or the tech buddy are not available and there is a problem. If the new joiner has a question he/she can always contact the recruiter.
+The person who recruited the new joiner. The Recruiter provides the overall structure for the onboarding process (Github tickets, main accounts) and is in general the fallback in case the buddy or the tech buddy are not available and there is a problem. If the new joiner has a question he/she can always contact the recruiter.
```

Comment:
```suggestion
The person who recruited the new joiner. The Recruiter provides the overall structure for the onboarding process (GitHub tickets, main accounts) and is in general the fallback in case the buddy or the tech buddy are not available and there is a problem. If the new joiner has a question he/she can always contact the recruiter.
```